hooks/useWindowDimensions.js

```diff
@@ -13,18 +15,21 @@ export default function useWindowDimensions() {
```

Comment:
In the `useWindowDimensions` function, you don't need to pass `hasWindow` to `getWindowDimensions` since it's already defined inside `useEffect`. The `hasWindow` check can just be used directly.